integration-cli/docker_cli_history_test.go

```diff
@@ -82,3 +83,52 @@ func TestHistoryNonExistentImage(t *testing.T) {
 	}
 	logDone("history - history on non-existent image must fail")
 }
+
+func TestHistoryImageWithComment(t *testing.T) {
+
+	// make a image through docker commit <container id> [ -m messages ]
+	runCmd := exec.Command(dockerBinary, "run", "-i", "-a", "stdin", "busybox", "echo", "foo")
```

Comment:
I'm assuming you specified `-a stdin` to avoid the echo messing with your command output. I don't think it's worth updating the PR, but it would be easier to have a trivial run such as:

    runCmd := exec.Command(dockerBinary, "run", "--name", "test", "busybox", "true")

And use the attributed name later on to refer to the container.